app/services/doc_auth/acuant/acuant_client.rb

```diff
@@ -57,9 +40,7 @@ def get_results(instance_id:)
       def post_images(
         front_image:,
         back_image:,
-        selfie_image:,
         image_source:,
-        liveness_checking_enabled: nil,
```

Comment:
My comment above applies here: https://github.com/18F/identity-idp/pull/7235

Reply:
ok i'll add them back in